base/Map.ts

```diff
@@ -1414,6 +1404,7 @@ export class Map {
             };
         } else {
             console.warn("Null NPC gotten...");
+            return () => {};
```

Comment:
Returning `null` here would have less overhead, no? If yes, I prefer returning `null`;

Reply:
Returning `null` runs the risk of breaking things that depend on having a function.

Reply:
Okay, I finally got a chance to look.  Returning `null` would actually break things as they are now, thanks to this:
```
                            if (get_djinn_fire_event !== undefined) {
                                get_djinn_fire_event();
                            }
```
Since `null !== undefined`, we would end-up calling `null()`, which would fail.
We can actually completely remove that check, though, since we're returning a dummy function -- I'll do that now.

Reply:
Done in a4eb2e700eb2dd70f8643ec601f328894b44e479.